tooling/minimize/tests/pass/ops.rs

```diff
@@ -5,12 +5,17 @@ fn black_box<T>(t: T) -> T { t }
 
 fn main() {
     print(-black_box(-42));
+    print(!black_box(-43));
```

Comment:
Please don't add more `print` tests -- it's so hard to figure out which line of output corresponds to which part of the test.

Instead, use `assert!`.